apps/meteor/client/hooks/useFormatNumber.ts

```diff
@@ -0,0 +1,12 @@
+import { useLanguage } from '@rocket.chat/ui-contexts';
+import { useCallback } from 'react';
+
+export const useFormatNumber = (options?: Intl.NumberFormatOptions) => {
```

Comment:
Where is this used?

Reply:
it is not used yet, but since we are introducing this pattern I already create the hook to help other